src/Nethermind/Nethermind.Evm/Precompiles/ModExpPrecompile.cs

```diff
@@ -91,7 +91,7 @@ private static long DataGasCostInternal(ReadOnlySpan<byte> extendedInput, ReadOn
         UInt256 exp = new(inputData.Span.SliceWithZeroPaddingEmptyOnError((int)startIndex, (int)expLengthUpTo32), true);
         UInt256 iterationCount = CalculateIterationCount(expLength, exp, releaseSpec.IsEip7883Enabled);
         bool overflow = UInt256.MultiplyOverflow(complexity, iterationCount, out UInt256 result);
-        result /= 3;
+        if (!releaseSpec.IsEip7883Enabled) result /= 3;
```

Comment:
Surrounding with `{ }` and new lines would look more in line with the current style